src/server/oembed.js

```diff
@@ -77,4 +85,15 @@ router.get('/oembed', cors(), async (req, res, next) => {
   }
 });
 
+
+function errorHandler(err, req, res, next) {
+  const msg = (err.response && err.response.data) || err.message || err;
+  if (err.name === 'ValidationError') {
+    return res.status(400).send(msg);
+  }
+
+  return res.sendStatus(500);
+}
+router.use(errorHandler);
```

Comment:
not sure if this should be put in `src/server.jsx` instead @nwingt

Reply:
@mileswong If this errorHandler is specific for oembed, I think it's okay to put in here, otherwise should put in `src/server.jsx`

Reply:
i think it's for oembed, and just keep the other parts remain unchanged